schema/readout-FakeCardReader-make.jsonnet

```diff
@@ -4,12 +4,16 @@
     queue: "fake-elink"
 
     // Make a conf object for cardreader
-    conf(lid=0, inplim=10485100, rate=166, rawtype="wib", datfile="/tmp/frames.bin", qtms=2000) :: {
+    conf(lid=0, inplim=10485100, rate=166, rawtype="wib", datfile="/tmp/frames.bin", qtms=2000,
+         raw_win_tp=true, tp_datfile="/tmp/tp_frames.bin", tp_rate=66) :: {
         link_ids: [lid],
         input_limit: inplim, 
         rate_khz: rate, 
         data_filename: datfile, 
-        queue_timeout_ms: qtms
+        queue_timeout_ms: qtms,
+        tp_enabled: raw_wib_tp,
```

Comment:
raw_wib_tp won't be found, as above the parameter is "raw_win_tp"?
Did you do schema generation?